doc/nrf/known_issues.rst

```diff
@@ -411,6 +437,14 @@ NCSDK-18376: P-GPS in external flash fails to inject first prediction during loa
   When using external flash with P-GPS, stored predictions cannot be reliably read to satisfy the modem's need for ephemeris data when actively downloading and storing predictions to the same external flash.
   Once the download is complete, the predictions can be reliably read.
 
+nRF7002
```

Comment:
Please add nRF7 as the main heading and nRf7002 under that?

Reply:
There is no other nRF7 SoC that would justify this. For now, I'll use the approach for nRF9160. If there are more SoCs from the 91 family or the nrf7 family, I'll add the respective headers.

Reply:
OK